server/src/main/scala/com/cloudera/livy/server/ApiVersioningSupport.scala

```diff
@@ -0,0 +1,98 @@
+package com.cloudera.livy.server
+
+import javax.servlet.http.HttpServletRequest
+
+import org.scalatra.{NotAcceptable, ScalatraBase}
+
+/**
+  * Livy's servlets can mix-in this trait to get API version support.
```

Comment:
There aren't many examples in the code base, but the existing docs use javadoc-style indentation. Would you mind using that? It looks slightly different:

    /**
     * The docs go here.
     */

Wonder if I can write a style checker for that.